zerver/views/streams.py

```diff
@@ -395,10 +391,6 @@ def get_streams_backend(request, user_profile,
                              include_all_active)
     return json_success({"streams": streams})
 
-def get_public_streams_backend(request, user_profile):
-    return get_streams_backend(request, user_profile, include_public=True,
-                               include_subscribed=False, include_all_active=False)
```

Comment:
We should keep the API default here, but the frontend should be fetching with `include_subscribed=False`.  This means that the commit switching the frontend should add the argument needed to preserve the correct behavior (in this case `include_subscribed=False`.